lib/nazrin/data_accessor/active_record.rb

```diff
@@ -5,7 +5,7 @@ class ActiveRecord < Nazrin::DataAccessor
       def load_all(ids)
         records_table = {}
         options.each do |k, v|
-          model = model.send(k, v)
+          @model = model.send(k, v)
```

Comment:
Sorry, bit late here, but is it an issue that this mutates the `@model` ivar? Or is a data accessor created for every request? One option for immutable `@model` is:

```ruby
relation = options.reduce(model) { |rel, (k, v)| rel.send(k, v) }

relation.where(id: ids).each do |record|
  # ...
```

Reply:
That makes sense...
That sounds great, `@model` must be immutable.